sqlvm-alwayson-cluster/README.md

```diff
@@ -46,3 +42,29 @@ This template creates the following resources:
 	+ 	Witness - Latest Windows Server 2012 R2 Image
 
 + 	Once deployed access can be gained by the public IP address of Primary Domain Controller
+
+## Deployment steps
+
+You can click the "deploy to Azure" button at the beginning of this document.
+
+|Name|Description|Example|
```

Comment:
Any template on Azure.com should have a table. Here's an example https://azure.microsoft.com/en-us/documentation/templates/bosh-setup/

Looking up expected parameters on Github is easier since the user has direct access to the source file.